server/db/models/models.spec.js

```diff
@@ -0,0 +1,136 @@
+
+const { expect } = require('chai')
+const db = require('../index');
+const Product = require('./product');
+const Review = require('./review');
+const LineItem = require('./lineItem');
+const Order = require('./order');
+const Category = require('./category');
+const Address = require('./address');
+
```

Comment:
All of these models are in the index file, so maybe could import

const { Product, Review, etc. } = require('./index')